pyQuARC/main.py

```diff
@@ -300,14 +302,20 @@ def display_results(self):
         parser.error(
             f"The given format is not supported. Only {', '.join(SUPPORTED_FORMATS)} are supported."
         )
+
+    if cmr_host := args.cmr_host:
+        if is_valid_cmr_url(cmr_host):
+            os.environ["CMR_URL"] = cmr_host
+        else:
+            raise Exception(f"The given CMR host is not valid: {cmr_host}")
```

Comment:
This is perfectly fine but with exceptions I usually prefer to write it as a negative:

```
if not is_valid_cmr_url(cmr_host):
    raise Exception(f"The given CMR host is not valid: {cmr_host}")
    
os.environ["CMR_URL"] = cmr_host
```

Also instead of a generic `Exception` we could subclass it into a `CMRURLNotValidException` with a custom error message. This way if we have other exceptions we need to handle in the future, they would just be additional `if not` statements.

Reply:
I agree, but since we don't have many other Exceptions right now and the flow doesn't differ for different exception types, we could possibly skip that right now, no?